src/main/java/com/nike/cerberus/archaius/client/ArchaiusCerberusUrlResolver.java

```diff
@@ -17,7 +17,7 @@
 package com.nike.cerberus.archaius.client;
 
 import com.amazonaws.regions.Regions;
-import javax.annotation.Nullable;
+import io.github.resilience4j.core.lang.Nullable;
```

Comment:
What's this?

Reply:
`javax.annotation.Nullable` does not exist in Java 8, so I replaced it with a different library:
https://github.com/resilience4j/resilience4j/blob/master/resilience4j-core/src/main/java/io/github/resilience4j/core/lang/Nullable.java